projects/packages/forms/src/blocks/contact-form/components/jetpack-form-notifications-settings.js

```diff
@@ -0,0 +1,107 @@
+import { FormTokenField, ToggleControl } from '@wordpress/components';
+import { store as coreStore } from '@wordpress/core-data';
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import InspectorHint from '../../shared/components/inspector-hint';
+
+const JetpackFormNotificationsSettings = ( { setAttributes, notificationRecipients } ) => {
```

Comment:
Can we make this less verbouse. We are already in the JetpackForm contexts here.